apps/src/feedback.js

```diff
@@ -987,6 +989,15 @@ FeedbackUtils.prototype.showSimpleDialog = function (Dialog, options) {
   if (options.bodyText) {
     contentDiv.innerHTML += '<p>' + options.bodyText + '</p>';
   }
+  var textBox;
+  if (options.prompt) {
+    textBox = document.createElement('input');
+    textBox.className = "name-textbox";
```

Comment:
Was using it for styling (adding a margin below the textbox). Switched to a style for `input` instead.